sharder/deterministic.go

```diff
@@ -11,16 +11,21 @@ import (
 	"sync"
 	"time"
 
+	"github.com/dgryski/go-wyhash"
 	"github.com/honeycombio/refinery/config"
 	"github.com/honeycombio/refinery/internal/peer"
 	"github.com/honeycombio/refinery/logger"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 )
 
-// shardingSalt is a random bit to make sure we don't shard the same as any
-// other sharding that uses the trace ID (eg deterministic sampling)
-const shardingSalt = "gf4LqTwcJ6PEj2vO"
+// These are random bits to make sure we differentiate between different
+// hash cases even if we use the same value (traceID).
+const (
+	shardingSalt        = "gf4LqTwcJ6PEj2vO"
+	peerSeed     uint64 = 6789531204236
+	traceSeed    uint64 = 5498616120353
```

Comment:
Is this used anywhere?

Reply:
Good catch; it was for a different variant of the algorithm but is not any longer.